inst/defaults/full/config/global.dcf

```diff
@@ -8,7 +8,7 @@ munging: TRUE
 logging: FALSE
 logging_level: INFO
 load_libraries: FALSE
-libraries: reshape, plyr, dplyr, ggplot2, stringr, lubridate
+libraries: reshape2, tidyverse, plyr, lubridate
```

Comment:
It is not recommended to load `plyr` after `dplyr` and raises a warning, please swap the two arround. Also, `stringr` is installed as a dependency for `tidyverse` but is not actually loaded automatically, so should not be removed.

Reply:
Thanks for the feedback. Making these changes.

Reply:
I'm also tempted to insert `forcats` in there. Thoughts? It's been extremely useful in re-coding factors, grouping factors etc.